app/src/main-process/app-window.ts

```diff
@@ -59,6 +60,10 @@ export class AppWindow {
       windowOptions.frame = false
     }
 
+    if (__LINUX__) {
+      windowOptions.icon = path.join(__dirname, 'icon-logo.png')
```

Comment:
I think you had this right previously with looking in the `static` subdirectory, like this:

```ts
windowOptions.icon = path.join(__dirname, 'static', 'icon-logo.png')
```

Here's what the app directory looks like after building:

```shellsession
$ pwd
/home/parallels/src/desktop/dist/desktop-linux-x64/resources/app
$ ls
ask-pass.js      crash.html    highlighter.js      node_modules     ui.css
ask-pass.js.map  crash.js      highlighter.js.map  package.json     ui.css.map
cli.js           crash.js.map  index.html          renderer.js      yarn.lock
cli.js.map       emoji         keytar.node         renderer.js.map
crash.css        emoji.json    main.js             runas.node
crash.css.map    git           main.js.map         static
$ ls static
ask-pass-trampoline.sh      empty-no-repo.svg
choosealicense.com          gitignore
default-avatar.png          icon-logo.png
empty-no-branches.svg       licenses.json
empty-no-commit.svg         welcome-illustration-left-bottom.svg
empty-no-file-selected.svg  welcome-illustration-left-top.svg
empty-no-pull-requests.svg  welcome-illustration-right.svg
```

But on my Ubuntu 16.04 VM I got a generic icon when running the app in development mode:

<img width="247"  src="https://user-images.githubusercontent.com/359239/34065068-54a4838c-e24e-11e7-953f-673086628589.png">

Same thing with the installed AppImage:

<img width="225" src="https://user-images.githubusercontent.com/359239/34065099-adf96c22-e24e-11e7-8a74-3093f8a22032.png">

Maybe there's something off with my setup?

Reply:
@shiftkey that's odd. When running yarn dev:build on my machine, a `icon-logo.png` file is being copied to `dist/desktop-linux-x64/resources/app/`. Are you building it a different way? I'm not that familiar with yarn yet, so there might be a different release task.

Anyway, changing it back to `static/icon-logo.png` works too, I've updated the PR.

Reply:
@picandocodigo I'm only doing `yarn build:dev` or `yarn build:prod`

Reply:
Just tried `yarn build:prod` too, and in both cases (prod and dev) for some reason the logo is being copied to both `/dist/desktop-linux-x64/resources/app` and `/dist/desktop-linux-x64/resources/app/static`.

If it doesn't happen in your VM, it might be an issue with my setup or something. The icon still shows up using `windowOptions.icon = path.join(__dirname, 'static', 'icon-logo.png') `. And I `rm -rf` the `dist` directory before every `yarn build`.

Reply:
I think this might have been some caching in the `out` directory to blame. We're not as aggressive with cleaning that directory up between builds - we should be nuking `dist`.